### ISSUE TITLE
PHPUnit faild with follwing error

### ISSUE THREAD

**Issue by nasirkhan** [myclabs/DeepCopy#119]
I installed a fresh laravel and tried to run PHPUnit and getting the following error. 

```
PHP Fatal error:  Cannot redeclare DeepCopy\deep_copy() (previously declared in /usr/share/php/DeepCopy/deep_copy.php:13) in /var/www/html/laravel/vendor/myclabs/deep-copy/src/DeepCopy/deep_copy.php on line 16
```

**Comment by theofidry** [comment]
This is happening because the file in which `deep_copy()` is being included twice. Since it's second time this issue is reported (cf. #102), I suspect there is something peculiar happening in Laravel... Could you provide more details?

- How you installed it?
- What dependencies do you have (`composer show`)
- Your PHPUnit config and how you ran it
- Is the repo accessible?

**Comment by nasirkhan** [comment]
Here is the steps to reproduce the issue, 

1. Install Laravel, you may follow the doc [here](https://laravel.com/docs/5.6) or just run `composer create-project --prefer-dist laravel/laravel blog`
2. go to the directory `blog`
3. run `phpunit`
4. You should get the error 

Laravel framework in on github https://github.com/laravel/laravel 

PHPUnit config is there as well https://github.com/laravel/laravel/blob/master/phpunit.xml

**Comment by theofidry** [comment]
Ah gotcha, I see the issue. It's a good old autoloading conflict: PHPUnit ships with DeepCopy so it's already loaded when you execute `phpunit` but in the tests you require the project autoload which also has DeepCopy. Unlike classes, functions are always autoloaded when registered to Composer (because of the lack of function autoloading). Well although it feels wrong we'll add a check so that the function is not registered twice

**Comment by ARKKYN** [comment]
please install  [phpunit 7.1](https://phpunit.readthedocs.io/en/7.1/installation.html ) and try again. that worked for me @nasirkhan

**Comment by theofidry** [comment]
@ARKKYN this shouldn't have much to do with the PHPUnit version, but rather with using PHPUnit globally whilst its already installed in the project

**Comment by nasirkhan** [comment]
@ARKKYN thanks. i just installed phpunit7.1 and it runs perfectly.

**Comment by ARKKYN** [comment]
@theofidry True, I just gave a temporary fix

**Comment by theofidry** [comment]
Closed by #120, should work fine now.
